app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ProfilesController < ApplicationController
+  def show
+    unless current_user.present?
```

Comment:
hmm to po co w ApplicationController ten authorize_user jak i tak chcesz tutaj to sprwadzac